src/actions/auth.js

```diff
@@ -15,54 +16,54 @@ export const types = {
   LOGOUT_FAILURE: 'LOGOUT_FAILURE',
 
   VERIFY_REQUEST: 'VERIFY_REQUEST',
-  VERIFY_SUCCESS: 'VERIFY_SUCCESS'
+  VERIFY_SUCCESS: 'VERIFY_SUCCESS',
```

Comment:
theres a setting for this in prettier. `Trailing Comma`, if you set this to none it won't add these in.